migrations/tables/app_status.yaml

```diff
@@ -21,3 +21,5 @@ spec:
         type: integer
       - name: sequence
         type: integer
+      - name: embeddedcluster_state
+        type: text
```

Comment:
it does feel weird to have the cluster state in the app status table, but in embedded installs, the cluster is a part of the application, so i think i'm good with it. curious to get other opinions on this one

Reply:
I would suggest creating a new table for the embeddedcluster status as it's a different from the app status and is updated/read separately. It's even shown separately in the UI. Similar to how we have preflight results/status in a separate table even though it's technically part of the app.

Reply:
it's been moved to its own table now